docs/messaging/README.md

```diff
@@ -11,6 +11,34 @@ path_base_for_github_subdir:
 
 This document defines semantic conventions for messaging systems spans, metrics and logs.
 
+> [!Warning]
+>
+> Existing messaging instrumentations that are using
+> [v1.24.0 of this document](https://github.com/open-telemetry/semantic-conventions/blob/v1.24.0/docs/messaging/messaging-spans.md)
+> (or prior):
+>
+> * SHOULD NOT change the version of the messaging conventions that they emit by default
+>   until the messaging semantic conventions are marked stable.
+>   Conventions include, but are not limited to, attributes,
+>   metric and span names, span kind and unit of measure.
+> * SHOULD introduce an environment variable `OTEL_SEMCONV_STABILITY_OPT_IN`
+>   in the existing major version which is a comma-separated list of values.
+>   If the list of values includes:
```

Comment:
looks like a typo in db also, I can send PR to fix that
```suggestion
>   The list of values includes:
```

Reply:
Ah good catch. I fixed in all files here.